MAINTAINERS.md

```diff
@@ -3,23 +3,17 @@
 
 libuv is currently managed by the following individuals:
 
-* **Anna Henningsen** ([@addaleax](https://github.com/addaleax))
```

Comment:
@addaleax I wasn't 100% sure about you but your last commit is from 2020 so... probably?